hnn_core/gui/gui.py

```diff
@@ -1187,17 +1332,17 @@ def add_connectivity_tab(params, connectivity_out,
                 # the connectivity list should be built on this level
                 receptor_related_conn = {}
                 for receptor in receptors:
-                    conn_indices = pick_connection(net=net,
+                    conn_indices = pick_connection(net=network,
```

Comment:
why rename to `network`? nitpick but by convention we use `net` throughout the code base

Reply:
renamed back to `net`